### PR TITLE
refactor: small simplifications 2

### PR DESCRIPTION
**Pull request by jly36963**
## Changes

- Remove shallow copy logic in `lib/yargs-factory.js`.
- Use ternary and || operator where practical.
- Update comments.
- Lint.

I'm still new at this, so let me know if there is anything I can do better.

### CODE REVIEW

File: lib/command.ts [hunk 1]
```diff
@@ -602,7 +596,7 @@ export class CommandInstance {
       });
 
       Object.keys(parsed.argv).forEach(key => {
-        if (positionalKeys.indexOf(key) !== -1) {
+        if (~positionalKeys.indexOf(key)) {
```
Comment (bcoe) [inline lib/command.ts:599]: I think here you could just use `.includes` now?

**Review by bcoe — COMMENTED** [review]
left one small nit.

File: lib/command.ts (hunk 1 above)
Comment (jly36963) [reply lib/command.ts:599]: Sounds good to me.

**Comment by jly36963** [comment]
Concede it is.
Thank you for always being kind -- I wish you the best :)

**Pull request closed without merging**